newIDE/app/src/UI/TwoStatesButton.js

```diff
@@ -0,0 +1,62 @@
+// @flow
+
+import * as React from 'react';
+import ButtonGroup from '@material-ui/core/ButtonGroup';
+import Button from '@material-ui/core/Button';
+import { dataObjectToProps } from '../Utils/HTMLDataset';
+import { focusButton } from './Button';
+
+type Props = {|
+  leftButton: {| label: React.Node, value: string, id?: string |},
+  rightButton: {| label: React.Node, value: string, id?: string |},
+  onChange: string => void,
+  value: string,
+  addDatasetEffective?: boolean,
```

Comment:
should this be always done by default, and remove the need of this boolean?

Reply:
I'm not sure about that. I could add it by default